fretboard/src/main/java/mozilla/components/service/fretboard/Sync.kt

```diff
@@ -0,0 +1,23 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package mozilla.components.service.fretboard
+
+import java.io.Serializable
+
+/**
+ * Helper class that represents a sync
+ * job in order to pass it to the scheduler
+ */
+data class Sync(
+    private val experimentSource: ExperimentSource,
+    private val experimentStorage: ExperimentStorage
+) : Serializable {
```

Comment:
Instead of this class I would prefer the job could access those things from a Fretboard object as mentioned above.